doc/RELEASE_NOTES.md

```diff
@@ -0,0 +1,828 @@
+
+All tools registered prior to installing `v0.71.0-alpha` must be re-registered; tools registered locally or remotely on a host with v0.69 or earlier of the `pbench-agent` will be ignored.
+
+
+## New Support for Prometheus and PCP-based Tools
+
+The new "Tool Meister" sub-system enables support of Prometheus and PCP-based tools for data collection.
+
+The existing tools supported prior to the v0.71 release can be categorized as "Transient" tools.  By _transient_ we mean that a given tool is started and stopped immediately around the execution of a benchmark workload.  For example, when using `pbench-fio -b 4,16,32 -t read,write`, the transient tools are started immediately before each `fio` job is executed, and stopped immediately following its completion, for each of the 6 (six) `fio` jobs that would be run.
+
+A new category is introduced for Prometheus and PCP called "Persistent" tools. Persistent are started once at the beginning of a benchmark script, stopped at its end.  Using the previous `pbench-fio` example, persistent tools would be started before any of the 6 (six) `pbench-fio` jobs begin, and would be stopped once all six end.
+
+When persistent tools are used, data is continuously collected from the data sources ("exporters", in the case of Prometheus, and "PMCDs", in the case of PCP) and stored local to the execution of the Tool Data Sink.
+
+Note that for transient tools, where data for the transient tool is collected locally on the host the tool is registered, the collected data is sent to the Tool Data Sink when the benchmark script deems it won't impact behavior of the benchmark itself.
+
+### Prometheus tools: `node-exporter` and `dcgm`
+
+Two new pbench "tools" have been added, `node-exporter` and `dcgm`.  If one registers either or both of these new tools (e.g. via `pbench-register-tools --name=node-exporter`), then the Tool Meister sub-system will run the `node_exporter` code on the registered hosts, and a local instance of Prometheus to collect the data.  The collected Prometheus data is stored in the pbench result directory as a tar ball at: `${pbench_run}/<script>_<config>_YYYY.MM.DDTHH.mm.ss/tools-<group>/prometheus`.
+
+For the duration of the run, the Prometheus instance is available on `localhost:9090` if one desires to review the metrics being collected live.
+
+_**NOTE WELL**_: like all the other "tools" the `pbench-agent` supports, the `node-exporter` and `dcgm` tools themselves need to be installed separately on the registered hosts.
+
+The new `dcgm` tool requires Python 2, an Nvidia based install which might conflict with the Pbench Agent's Python 3 operational requirement in some cases.
```

Comment:
I don't think that these should be separate paragraphs.

s/Nvidia based/Nvidia-based/ ?

The warning that the Python installations might conflict is so vague as to be almost useless.  For instance, there is no description of how the user might recognize that a conflict has occurred, and there is no indication of what the user should do to overcome the conflict.

Reply:
I think we should drop this entirely.  Since this dcgm work was done, the latest implementation of the `dcgm-exporter` appears to be written in Go.

Reply:
Sounds OK...so long as anyone installing `dcgm-exporter` doesn't do something silly like install the old Python-based one...but, perhaps it should be obvious how silly that is.